### ISSUE TITLE
Correct /version in NginX conf

### ISSUE THREAD

**Issue by drasko** [absmach/magistrala#396]
This https://github.com/mainflux/mainflux/blob/master/docker/nginx.conf#L156 maps `/version` directly - which is wrong as it is non-existent API endpoint.

Probably should be pre-fixed with something like `<service>/version`.

**Comment by anovakovic01** [comment]
Do we really need this to be exposed? IMHO this is internal endpoint that should be used for service health check or ping.

**Comment by anovakovic01** [comment]
@drasko I agree, but do you really need to expose internal structure to the outside world? If this nginx config is used for dev, then maybe we shouldn't enforce TLS.

**Comment by chombium** [comment]
@drasko quick question: do we want to check only if nginx is alive (simple get request) or we also want to have some stats?

I found few interesting links and projects which expose/export the nginx metrics to Prometheus, they need some configuring, though:
https://hnlq715.github.io/nginx-vts-exporter/
https://github.com/nginxinc/nginx-prometheus-exporter
https://sysdig.com/blog/monitor-nginx-kubernetes/
https://sysdig.com/blog/monitor-nginx-kubernetes-metrics-alerts/

It might be a bit of overkill to setup all this, but we should start thinking about monitoring the infrastructure components (nginx, redis, nats...) as well ;)

**Comment by chombium** [comment]
@anovakovic01 sure, it does not worth to invest the time in it if we do it in the near feature. For now we could simply enable the  "stub_status" module in nginx and who ever needs the stats can use them.

Is there some timeline about Ambassador and when are doing the switch?

**Comment by anovakovic01** [comment]
It's work in progress. I should send PR in next few days.

**Comment by drasko** [comment]
@chombium I had in mind a simple ping for now, just health check. I was writing [this blog](https://medium.com/mainflux-iot-platform/deploy-mainflux-on-digital-ocean-in-3-simple-steps-973f99470f72) and I realized that a developer will not have a simple way to check from outside if the instance on DO is running OK.

Then I looked at `nginx` config, because I know that we have `/version` healthecks on services and I saw this bug :).
